packages/react/src/themes/teams/components/Tooltip/tooltipContentStyles.ts

```diff
@@ -11,19 +26,22 @@ const tooltipContentStyles: ComponentSlotStylesInput<
     borderRadius: v.borderRadius,
     display: 'block',
     maxWidth: v.maxWidth,
-
-    ...(p.pointing && getPointerStyles(v.pointerOffset, v.pointerMargin, rtl, p.placement).root),
+    color: v.color,
+    background: v.backgroundColor,
+    ...(p.pointing &&
+      getPointerStyles(getPointerOffset(p.placement, v), v.pointerMargin, rtl, p.placement, true)
```

Comment:
I would suggest to calculate pointer styles once and store them, given how long list of `getPointerStyles` arguments is. Something like

```jsx
// here we will be able to capture semantics of these styles in variable name
const svgPointerStyles = getPoiterStyles(...)

return {
  root: () => ({
    ...,
    ...svgPointerStyles.root
  }),
  pointer: () => ({
    ...,
    ...svgPointerStyles.pointer
  })
}
```

Reply:
I cannot extract this in the beginning, because the ComponentSlotStylesInput is an object, not a function, but both in the root and pointer, I can extract this as a variable in the beginning... Would that be enough?